README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
Unless we want to create a tag for ES6 before any ES7 tests are checked in (and thus not include many new ES6 tests in the ES6 tag), this would work, otherwise we will require branches and integration thereof.